spec/system/comments/user_fills_out_comment_spec.rb

```diff
@@ -116,6 +117,21 @@
     end
   end
 
+  context "with TwitterTimeline tag" do
+    before do
+      visit article.path.to_s
+
+      wait_for_javascript
+    end
+
+    it "User fill out comment box with a TwitterTimeline tag, then clicks preview" do
+      fill_in "text-area", with: twitter_comment
+      click_button("Preview")
+
+      expect(page).to have_css(".ltag-twitter-timeline-body iframe", count: 1)
```

Comment:
Hey @akashdotsrivastava. When I pulled your branch down, this spec keeps failing. Specifically, the output states that it cannot find the element with CSS `.ltag-twitter-timeline-body iframe`.

It may be just on my end though 🤷🏾‍♀️

Reply:
@msarit I feel you. I always have trouble with system tests locally, that's why I cannot wait for us to switch to E2E for all of these. That said, I tried to run `bundle exec rspec spec/system/comments/user_fills_out_comment_spec.rb` and it doesn't pass for me either

Reply:
Hey @msarit and @rhymes . Thanks for your comments. I went back and double-checked the above spec. I ran it as a solitary as well the whole suite. Passed in both the cases.

Is there something I can do to make sure it works more convincingly? I mean it is more or less a copy of a similar test that tests runkit tag on preview and checks if the runkit iframe exists, only the same for Twitter in this case. I see it passing though in the build https://travis-ci.com/github/forem/forem/jobs/479982271

Reply:
Nope, I trust your code and CI. Thank you for clarifying!